packages/loot-core/src/server/preferences/app.ts

```
@@ -18,4 +18,17 @@ const savePreferences = async ({
   await db.update('preferences', { id, value });
 };
 
+const getPreferences = async (): Promise<SyncedPrefs> => {
+  const prefs = (await db.all('SELECT id, value FROM preferences')) as Array<{
+    id: string;
+    value: string;
+  }>;
+
+  return prefs.reduce<SyncedPrefs>(
+    (carry, { value, id }) => ({ ...carry, [id]: value }),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)